_data/packages.yml

```diff
@@ -2,11 +2,17 @@
   package_description: A spectral-timing software package for astrophysical X-ray
       (and other) data
   submitting_author:
-      name: Name
+      name: Matteo Bachetti
```

Comment:
Ok so I think we need a new issue about this, specifically in pyos meta. what i'm seeing here is our code isn't processing multiple maintainers correctly. We will need to debug pyosmeta to see why.